libioc/Config/Jail/File/__init__.py

```diff
@@ -30,36 +30,6 @@
 import libioc.helpers_object
 import libioc.LaunchableResource
 
-
-class ResourceConfig:
-    """Shared abstract code between various config files in a resource."""
```

Comment:
The methods were moved to Resource itself. A resource should be able to check that a file is real relative to its storage location rather than File abstractions having this feature. You are right, the methods should be public.